tests/ops/test_cv_ops.py

```diff
@@ -188,15 +188,15 @@ def test_heisenberg_transformation_nongaussian(self):
         Heisenberg transformation of non-Gaussian gates."""
         op = cv.Kerr
         with pytest.raises(RuntimeError, match=r"not a Gaussian operation"):
-            op_ = op(*[0.1] * op.num_params, wires=[0] * op.num_wires)
+            op_ = op(*[0.1] * op.num_params, wires=range(op.num_wires))
```

Comment:
I'm not sure what the reason for ``[0]*op.num_wires`` was, but it is an illegal wires argument since it repeats the zero. Maybe this was just not important for the error check here, but now it is.